src/plugins/data_source/server/plugin.ts

```diff
@@ -56,9 +56,11 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
       config
     );
 
+    const endpointDeniedIPs = config.endpointDeniedIPs;
     const dataSourceSavedObjectsClientWrapper = new DataSourceSavedObjectsClientWrapper(
       cryptographyServiceSetup,
-      this.logger.get('data-source-saved-objects-client-wrapper-factory')
+      this.logger.get('data-source-saved-objects-client-wrapper-factory'),
+      endpointDeniedIPs
```

Comment:
nit: can we directly use `config.endpointDeniedIPs` here and avoid extra const?

Reply:
I prefer using the explicit input for clear dependency purpose.